package/MDAnalysis/lib/util.py

```diff
@@ -207,8 +207,11 @@
 import inspect
 
 from ..exceptions import StreamWarning, DuplicateWarning
-from ._cutil import unique_int_1d
-
+try:
+    from ._cutil import unique_int_1d
+except ImportError:
+    print("MDAnalysis not installed properly.")
+    raise
```

Comment:
I'm pretty sure you'd want to raise an `ImportError` rather than a generic `raise`? Otherwise, capturing it might broadly capture all other Exception classes as well.